packages/cli/src/utils.ts

```diff
@@ -153,6 +154,183 @@ export const validFieldsFromPkg = {
     }
     return obj;
   },
+  exports(
+    pkg: Package,
+    hasModuleBuild: boolean,
+    hasBrowserField: boolean,
+    hasWorkerField: boolean,
+    entrypointName: string,
+    forceStrategy?: DistFilenameStrategy
+  ): Record<string, ExportsConditions | string> {
+    let output: Record<string, ExportsConditions> = {};
+    let obj: ExportsConditions = exportsHelpers.root(
+      pkg,
+      hasModuleBuild,
+      entrypointName,
+      forceStrategy
+    );
+    if (hasBrowserField) {
+      obj = {
+        browser: exportsHelpers.target(
+          pkg,
+          hasModuleBuild,
+          entrypointName,
+          forceStrategy,
+          "browser"
+        ),
+        ...obj,
+      };
+    }
+    if (hasWorkerField) {
+      obj = {
+        worker: exportsHelpers.target(
+          pkg,
+          hasModuleBuild,
+          entrypointName,
+          forceStrategy,
+          "worker"
+        ),
+        ...obj,
+      };
+    }
+    output["."] = obj;
+    pkg.entrypoints.forEach((entrypoint) => {
+      if (entrypointName === entrypoint.name) {
+        return;
+      }
```

Comment:
`output["."]` won't necessarily exist

Reply:
I've made the change so that the `"."` condition is only created if needed (ie the package name matches the entrypoint name).